Dockerfile

```
@@ -1,3 +1,7 @@
-FROM scratch
-COPY gcp-artifact-registry-docker-proxy /
-ENTRYPOINT ["/gcp-artifact-registry-docker-proxy"]
+FROM alpine
+
+RUN apk --no-cache add curl
```

I think you can install just the certificates by adding [ca-certificates](https://pkgs.alpinelinux.org/package/edge/main/x86/ca-certificates)

```suggestion
RUN apk --no-cache add ca-certificates
```

---

Ah yeah, was trying to build this locally but Docker is acting up and forgot to push this up.

---

No worries!